### ISSUE TITLE
question: Applicability to Validating Causal Inference via Influence Functions

### ISSUE THREAD

**Issue by matthewvowels1** [ahmedmalaa/torch-influence-functions#1]
Dear Ahmed,

Many thanks for releasing this helpful code and for your general contributions to the field!

I had two quick questions about adapting it to reproduce your results from your paper on IFs for validating causal inference models...  

Firstly, am I correct in thinking that in that paper, the influence functions were taken w.r.t. model's estimated distributions (theta = {mu, pi}), rather than the *weights* of the neural networks? i.e. could you confirm what exactly theta is, in \dot{l}_{\theta}?  

Secondly, assuming the process for calculating the IFs is different for that paper to that which is presented in this repo, do you have any plans to release code for the causal validation paper?

All the best!
Matthew

**Comment by ahmedmalaa** [comment]
Hi Matthew, 

Thanks for reaching out. This code implements influence functions for the effect of training data points on the parameters of a neural net. It's a very different setup than the causal model validation problem where the IF estimates the model performance under missing "labels". The code for causal model evaluation will be out this summer on my Github account.

Thanks!

**Comment by matthewvowels1** [comment]
Dear Ahmed,

Many thanks for your response!

Ah yes, I thought as much, thank you for clarifying.  I will greatly look forward to its release! : ]

Matthew
